homeassistant/components/garage_door/rpi_gpio.py

```diff
@@ -25,7 +27,9 @@
         vol.Schema({
             'name': str,
             'relay_pin': int,
+            'relay_time': int,
             'state_pin': int,
+            'state_pull_mode': str,
```

Comment:
Can pull mode be something else than UP or DOWN? 
Is it a required value? Should it be optional and default to DEFAULT_PULL_MODE?